.github/workflows/automatic-updates.yml

```diff
@@ -39,6 +39,11 @@ jobs:
           GH_API_TOKEN: ${{ secrets.GH_API_TOKEN }}
         run: node --experimental-fetch check-pr-status.mjs ${{ github.repository }} ${{ steps.cpr.outputs.pull-request-number }}
 
+      - uses: hmarr/auto-approve-action@v2
```

Comment:
why this instead of https://github.com/juliangruber/approve-pull-request-action (from same person as the merge below)?

Also, the example from `peter-evans/create-pull-request` has an `if`  - any reason not to include that here?

Reply:
Honestly, I didn't notice there were an approve pull request action made by juliangruber. Will change.

We don't need an if because the script above will fail with exit code 1 if the checks don't pass, so the auto-approve-action won't run.

Reply:
cool 👍